Prototype/Components/list/list.html

```diff
@@ -0,0 +1,173 @@
+<div class="list list-theme-light">
```

Comment:
@Satyam1RP Make one theme variation by default in case list-theme-dark update styles